.github/workflows/daily-qa.yml

```diff
@@ -27,23 +27,27 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       latest-version: ${{ env.LATEST_VERSION }}
-      previous-latest-version: ${{ env.PREVIOUS_LATEST_VERSION }}
-      previous-previous-latest-version: ${{ env.PREVIOUS_PREVIOUS_LATEST_VERSION }}
+      second-last-version: ${{ env.SECOND_LAST_VERSION }}
+      third-last-version: ${{ env.THIRD_LAST_VERSION }}
+      fourth-last-version: ${{ env.FOURTH_LAST_VERSION }}
```

Comment:
:thought_balloon: There's a good chance that we will support 6 versions soon. We don't have to find a perfect solution now but long-term we might want to reduce some duplication here

Reply:
Step by step